.travis.yml

```diff
@@ -22,7 +22,7 @@ matrix:
     - os: linux
       sudo: required
       group: deprecated-2017Q3
-      go: 1.9
+      go: 1.9.3
```

Comment:
oh.. could you please add quotes around the version to conform with a travis/go version best practice ( https://docs.travis-ci.com/user/languages/go/).  thanks!
> Note that, in order to choose Go 1.10, you must use go: "1.10" (a string), not go: 1.10 (a float). Using a float results in the use of Go 1.1.